.babelrc.js

```diff
@@ -34,24 +34,30 @@ let envOpts = {
   loose: true
 };
 
-module.exports = {
+const config = {
   comments: false,
   presets: [
     ["@babel/env", envOpts],
-    "@babel/stage-0",
     "@babel/flow"
   ],
-  env: {
-    cov: {
-      auxiliaryCommentBefore: "istanbul ignore next",
-      plugins: [istanbulHacks]
-    }
-  }
+  plugins: [
```

Comment:
do we want to be explicit here? (also we only use export-namespace and numeric-separator once but shrug)

Reply:
Definitely.

There is an argument for using stage-0 as dogfooding but I don't think it's a good idea to dogfood something that is _supposed_ to be unstable and subject to change without notice.